src/leap/mail/imap/mailbox.py

```diff
@@ -500,9 +500,9 @@ def fetch(self, messages_asked, uid):
         is_sequence = True if uid == 0 else False
 
         # XXX DEBUG ---  if you attempt to use the `getmail` utility under
-        # imap/tests, it will choke until we implement sequence numbers. This
-        # is an easy hack meanwhile.
-        # is_sequence = False
+        # imap/tests, or muas like mutt, it will choke until we implement
+        # sequence numbers. This is an easy hack meanwhile.
+        is_sequence = False
```

Comment:
I see your point, but I also would like to avoid having to lookup what's the meaning of the passed uid == 0 when implementing sequences.

Right now, I'd rather leave it set to false to allow displaying of mail in mutt as a way to cross-check rendering of messages.

Having the first assignment commented is ok for you?